cypress/fixtures/plugin/chat.ts

```diff
@@ -0,0 +1,60 @@
+import { Chat } from '../../../dist'
+
+import fixtureChat from '../chat.json'
+
+export default function chat (): void {
+  // Messages
+  Chat.listen(message => {
+    if (message.content === fixtureChat.listen.listened) {
+      Chat.broadcast({
+        text: fixtureChat.listen.broadcasted
+      })
+    }
+  })
+  Chat.listen(message => {
+    if (message.content === fixtureChat.listen.listened) {
+      Chat.broadcast({
+        text: fixtureChat.listen.broadcasted
+      })
+    }
+  })
+
+  const re = new RegExp(fixtureChat.intercept.regex)
+  Chat.intercept(re, () => {
+    //
+  })
+
+  // Not enough stable to be tested (race condition)
+  // Chat.broadcast({
+  //   text: fixtureChat.broadcast.text
+  // })
+  // Chat.broadcast({
+  //   html: fixtureChat.broadcast.html
+  // })
+
+  // Not enough stable to be tested (race condition)
+  // Chat.send({
+  //   user: fixtureChat.send.user,
+  //   text: fixtureChat.send.text
+  // })
+  // Chat.send({
+  //   user: fixtureChat.send.user,
+  //   html: fixtureChat.send.html
+  // })
+
```

Comment:
Shouldnt it be ok now ?

Reply:
In a dedicated PR: https://github.com/livestorm/livestorm-plugin/pull/19 because it depends also on a main app fix.